indexES.mjs

```diff
@@ -0,0 +1,51 @@
+import mongoose from 'mongoose';
+// const mongoose = require("mongoose");
+
+// Import of the model Recipe from './models/Recipe.model.js'
+// const Recipe = require('./models/Recipe.model');
+import Recipe from './models/Recipe.model.js';
+// import Recipe from './models/Recipe.model';
+// import Recipe from './models/Recipe.model';
+// Import of the data from './data.json'
+// const data = require('./data.json');
+import data from './data.json' assert { type: 'json' };
+const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+
+// await mongoose.connect('mongodb://localhost:27017/pokemons-v3');
+// await mongoose.connection.db.dropDatabase();
+// await mongoose.connection.close();
+
+// await mongoose.connect('mongodb://localhost:27017/pokemons-v3');
+
+async function main() {
```

Comment:
no need for a async main function in a .mjs file -> top level await is a thing here :)